### ISSUE TITLE
Is there a easy way to rotate the cube without moving the world

### ISSUE THREAD

**Issue by silentrob** [larspetrus/Roofpig#11]
Hi Lars, This is such a great project!

So right now when you rotate the cube (using the key events) `CSXD` you are moving the camera and thus the position of your hands stay the same. So in terms of the algorithm U becomes R and then B etc.

It would be nice to rotate the cube and update your hand positions too. Is there an easy way you would recommend that?

**Comment by larspetrus** [comment]
Yeah, the keyboard controls are useless right now. Just a proof of concept, really.

Are you saying you'd want to try making it better?

**Comment by silentrob** [comment]
Ya I forked it and remapped MLRU to align with the general algorithms. But got stuck on rotating and was just looking for a clean way forward. I was envisioning a interactive mode. Where I can reset back to F2L solved and keep re-testing a random the OLL algorithms. But you could imagine the same thing for practicing solving F2L or PLL.

**Comment by silentrob** [comment]
So one year later - does `unhand()` help us here?

**Comment by larspetrus** [comment]
All it does right now is translate "hand notation" (`r U2 R U' r2' D r U R' U' r' D' r2 U' r'`) to "cube notation" (`L F2 R F' L2 F L U R' U' L' F' L2 F' L'`).

That seems related to what you're doing, but probably not the full solution. The underlying code could make it a lot easier though.

One reason it's hard for me to answer is that I'm not sure _exactly_ what you're trying to do.
